tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -28,24 +29,36 @@
-        </pre>
+        <div class="code-example-section">
+          <a href="https://www.tensorflow.org/api_docs/python/tf/Session" target="_blank">tf.Session</a>s:
+          <pre class="code-snippet" style='color:#000020;background:#f6f8ff;'><span style='color:#200080; font-weight:bold; '>from</span> tensorflow<span style='color:#308080; '>.</span>python <span style='color:#200080; font-weight:bold; '>import</span> debug <span style='color:#200080; font-weight:bold; '>as</span> tf_debug
+
+sess <span style='color:#308080; '>=</span> tf<span style='color:#308080; '>.</span>Session<span style='color:#308080; '>(</span><span style='color:#308080; '>)</span>
+sess <span style='color:#308080; '>=</span> tf_debug<span style='color:#308080; '>.</span>TensorBoardDebugWrapperSession<span style='color:#308080; '>(</span>sess<span style='color:#308080; '>,</span> <span style='color:#1060b6; '>"[[_host]]:[[_port]]"</span><span style='color:#308080; '>)</span>
+sess<span style='color:#308080; '>.</span>run<span style='color:#308080; '>(</span>my_fetches<span style='color:#308080; '>)</span></pre>
+        </div>
+        <div  class="code-example-section">
+          <a href="https://www.tensorflow.org/programmers_guide/estimators" target="_blank">Estimator</a>s
+          and
+          <a href="https://www.tensorflow.org/api_docs/python/tf/train/MonitoredSession" target="_blank">MonitoredSession</a>s:
+          <pre class="code-snippet" style='color:#000020;background:#f6f8ff;'><span style='color:#200080; font-weight:bold; '>from</span> tensorflow<span style='color:#308080; '>.</span>python <span style='color:#200080; font-weight:bold; '>import</span> debug <span style='color:#200080; font-weight:bold; '>as</span> tf_debug
+
+hook <span style='color:#308080; '>=</span> tf_debug<span style='color:#308080; '>.</span>TensorBoardDebugHook<span style='color:#308080; '>(</span><span style='color:#1060b6; '>"[[_host]]:[[_port]]"</span><span style='color:#308080; '>)</span>
+my_estimator<span style='color:#308080; '>.</span>fit<span style='color:#308080; '>(</span>x<span style='color:#308080; '>=</span>x_data<span style='color:#308080; '>,</span> y<span style='color:#308080; '>=</span>y_data<span style='color:#308080; '>,</span> steps<span style='color:#308080; '>=</span><span style='color:#008c00; '>1000</span><span style='color:#308080; '>,</span> monitors<span style='color:#308080; '>=</span><span style='color:#308080; '>[</span>hook<span style='color:#308080; '>]</span><span style='color:#308080; '>)</span></pre>
+        </div>
+        <div class="code-example-section">
+          <a href="https://www.tensorflow.org/api_docs/python/tf/keras/models" target="_blank">Keras Model</a>s:
+          <pre class="code-snippet" style='color:#000020;background:#f6f8ff;'><span style='color:#200080; font-weight:bold; '>import</span> tensorflow <span style='color:#200080; font-weight:bold; '>as</span> tf
```

Comment:
This snippet includes `import tensorflow as tf` but the first one for tf.Session doesn't.  Probably best if they're consistent.